packages/devextreme-cli/commands/themebuider.js

```diff
@@ -111,13 +111,11 @@ const installThemeBuilder = async version => {
         fs.copyFileSync(npmrc, installationNpmrc);
     }
 
-    await runCommand('npm', [
-        'install',
-        '--no-save',
-        `devextreme-themebuilder@${version}`
-    ], {
+    await packageManager.installPackage(`devextreme-themebuilder@${version}`, {
```

Comment:
We try to get version from package.json or node_modules. Should we change this behaviour?

Reply:
Fixed. Now we try to get the version from package-lock or yarn.lock